test/integration/server.h

```diff
@@ -294,6 +295,11 @@ class TestIsolatedStoreImpl : public StoreRoot {
     Thread::LockGuard lock(lock_);
     store_.forEachTextReadout(f_size, f_stat);
   }
+  void forEachScope(std::function<void(std::size_t)> f_size,
```

Comment:
Is this tested anywhere?

Reply:
No, but it's just a required override for a test impl of an interface. Neither are the other forEach methods above, from what I can tell.